ex5/5.css

```diff
@@ -0,0 +1,657 @@
+*{
+  margin: 0px;
+  padding: 0px;
+  color: black;
+  text-decoration: none;
+  font-size: 14px;
+  font-family: Cambria;
+  list-style-type: none;
+}
+
+.one{
+  width: 1200px;
+  margin: 0px auto;
+  display: flex;
+  height: 40px;
+  border-bottom: 1px solid #c0c0c0 ;
+  color: #313131;
+}
+.one .email{
+  display: flex;
+  line-height: 40px;
+}
+.one .email p{
+  padding: 0px 10px;
+}
+.one .email p:first-child{
+  padding-left: 0px;
+}
+.one .user {
+  display: flex;
+  line-height: 40px;
+  margin-left: 416px;
+}
+.one .user p{
+  padding: 0px 10px;
+}
+.one .user p:last-child{
+  padding-right: 0px;
+}
+#place{
+  line-height: 40px;
+  font-size: 14px;
+  vertical-align:bottom;
+}
+#diadiem{
+  line-height: 40px;
+  font-size: 17px;
+  vertical-align: bottom;
+}
+#dangnhap, #dangky{
+  line-height: 40px;
+  font-size: 16px;
+  vertical-align: bottom;
+  margin-right: 5px;
+}
+#dangky{
+  font-size: 10px;
+}
+
+.two{
+  display: flex;
+  width: 1200px;
+  margin: 0px auto;
+  margin-top: 20px;
+}
+.two a img{
+  height: 75px;
+  width: 250px;
+  margin-left: 30px;
+  margin-top: 5px;
+}
+.two #danhmuc{
```

Comment:
thường mấy cái thẻ to bao ngoài người ta hay đặt #id hơn ý

Reply:
oki a ơi